packages/common/mui/src/lib/simple-toggle/simple-toggle.tsx

```diff
@@ -59,7 +59,7 @@ export const SimpleToggle = ({
 const Container = styled(Box)(
   ({ theme }) => `
   height: 100%;
-  background: ${theme.palette.background.paper};
+  background: ${theme.palette.background.button};
```

Comment:
@anazari96 This toggle is used in a few other places inside table headers so the color changes here make it look wrong:

Your Branch:
![image](https://github.com/user-attachments/assets/f5831a17-6dcf-4e8c-adc4-7dfb08e42f32)

On Dev:
![image](https://github.com/user-attachments/assets/d7ddad1a-ac7a-4f72-8725-8342ca4202ca)

Reply:
Also on dark mode the text should be white:
![image](https://github.com/user-attachments/assets/39f78577-bc94-4ee5-a65d-d6e31d348e49)

It looks like it is now darker on your version:
![image](https://github.com/user-attachments/assets/171e5c58-f285-480a-b6c9-59a807f41c79)

Reply:
You can access dark mode via the settings:
![image](https://github.com/user-attachments/assets/3b99a01f-6836-49c8-b391-cda77035769e)